CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+# v1.13.0 (2022-03-??)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to release ([#2829])
+* Add `NoOpMigration` for migrations and migration-helpers
+  for when no data migration is necessary _(Also cherry-picked to v1.12.0)_ ([#2749])
+* Add info logging to migrator in order to document onging migrations ([#2751])
+* Rename `ncurses` to `libncurses` ([#2769])
+* Add `CAP_SYS_MODULE` and `CAP_CHROOT` to bootstrap containers ([#2772])
+* Upgrade `tokio` to latest LTS in Bottlerocket sources and "latest" in tools ([#2736])
```

Comment:
This seems like an internal detail too. I guess I'm unclear - is the target of this CHANGELOG users of Bottlerocket or developers of Bottlerocket. If just users, then I think there are a few things in here that may just be confusing and unnecessary to someone just trying to run Bottlerocket.